front/src/applications/stdcmV2/utils/checkStdcmConfigErrors.ts

```diff
@@ -16,6 +21,10 @@ const checkStdcmConfigErrors = (
     return { errorType: StdcmConfigErrorTypes.PATHFINDING_FAILED };
   }
 
+  if (pathSteps.some((step) => 'uic' in step && step.uic === -1)) {
```

Comment:
We should be not compact `pathSteps` when passing it to the function (and change the type for the parameter) to handle case like this : 

![Capture d’écran 2024-10-11 à 16 49 47](https://github.com/user-attachments/assets/0fb1b617-1cad-49bd-9684-bad7c41cc078)

```suggestion
  if (pathSteps.some((step) => step === null || 'uic' in step && step.uic === -1)) {
```

Reply:
Done :+1: